### PR TITLE
Release 53

### PR DESCRIPTION
**Pull request by CloCkWeRX**

### CODE REVIEW

**Comment by cesy** [comment]
Are we happy enough with this to push it to prod, or want to do some proper checks first?

**Comment by cesy** [comment]
@CloCkWeRX are you happy to change this from draft to ready to go, or what tests do we need to do first? Would be great to get all these security improvements onto prod.

**Comment by CloCkWeRX** [comment]
Oops, missed your comment @cesy! Have shifted it to "ready"

File: .devcontainer/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Ruby version
+VARIANT=3.1.3
```
Comment (CloCkWeRX) [inline .devcontainer/.env]:
```suggestion
VARIANT=3.1.4
```
For the dev container?

**Review by cesy — APPROVED** [review]

**Pull request merged**